lib/AArch64Base.ml

```diff
@@ -2059,6 +2177,25 @@ let do_pp_instruction m =
       sprintf "ADDVL %s,%s,%s" (pp_xreg r1) (pp_xreg r2) (m.pp_k k1)
   | I_CNT_INC_SVE (op,r,pat,k) ->
       pp_cnt_inc op r pat k
+(* Scalable Matrix Extention *)
+  | I_LD1SPT (v,r,ri,k,p,ra,idx) ->
+      sprintf "LD1%s {%s[%s,%s]},%s,%s" (pp_simd_variant v) (pp_zareg r) (pp_wreg ri) (m.pp_k k) (pp_preg p) (pp_addr ra idx)
```

Comment:
Hi @murzinv. The function `do_pp_instruction`  has a polymorphic type,  more precisely `'a basic_pp -> 'a kinstruction -> string`, where `'a` is the type of constant `k`.

 This permits using the same function for different tasks. In particular `'a` can be `int` as in `dump_instruction` below or `MetaConst.t` as in `dump_parsedIntruction`. One possible option to avoid the leading `#` for some constant arguments such as `offs` would be to add a `pp_k_nohash` member to the `basic_pp` record type and to to replace some calls to `m.pp_k` by calls to `m.pp_k_no_hash`. Of course `pp_k_n_hash` would not add the leading `#`.